decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/_promoted_process.html.erb

```diff
@@ -1,39 +1 @@
-<div class="card card--full card--process">
-  <div class="row collapse card--process__row">
-    <div class="columns mediumlarge-8 large-6 card--process__column">
-      <div class="card__content">
-        <%= link_to participatory_process_path(promoted_process), class: "card__link" do %>
-          <h2 class="card__title"><%= decidim_html_escape(translated_attribute(promoted_process.title)).html_safe %></h2>
-        <% end %>
-        <%= decidim_sanitize_editor html_truncate(translated_attribute(promoted_process.short_description), length: 630, separator: "...") %>
-        <%= link_to participatory_process_path(promoted_process), class: "button small hollow card__button" do %>
-          <span class="show-for-sr"><%= decidim_html_escape(translated_attribute(promoted_process.title)) %></span>
-          <%= t("participatory_processes.promoted_process.more_info", scope: "layouts.decidim") %>
-        <% end %>
-      </div>
-    </div>
-    <div class="columns mediumlarge-8 large-6 card--process__column">
-      <div class="card--full__image"
-           style="background-image:url('<%= promoted_process.attached_uploader(:hero_image).path %>')">
-        <div class="card__content row collapse">
-          <div class="large-6 large-offset-6 columns">
-            <%= link_to participatory_process_cta_path(promoted_process), class: "button expanded button--sc" do %>
-              <% if translated_in_current_locale?(promoted_process.active_step&.cta_text) %>
-                <span class="show-for-sr"><%= translated_attribute(promoted_process.active_step.cta_text) %></span>
-                <span aria-hidden="true"><%= translated_attribute(promoted_process.active_step.cta_text) %></span>
-              <% else %>
-                <span class="show-for-sr"><%= t(promoted_process.cta_button_text_key_accessible, resource_name: translated_attribute(promoted_process.title), scope: "layouts.decidim.participatory_processes.promoted_process") %></span>
-                <span aria-hidden="true"><%= t(promoted_process.cta_button_text_key, scope: "layouts.decidim.participatory_processes.promoted_process") %></span>
-              <% end %>
-              <% if promoted_process.active_step %>
-                <span class="button__info">
-                  <%= t("participatory_processes.promoted_process.active_step", scope: "layouts.decidim") %> <%= translated_attribute promoted_process.active_step.title %>
-                </span>
-              <% end %>
-            <% end %>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+<%= card_for promoted_process, highlight: true, size: :g %>
```

Comment:
I could not find any partial reference to it. Maybe just remove it ?

Reply:
In this case the partial path was resolved with `render_highlighted_partial_for`, but I've done the simplification similar to other parts and removed the use of partials and this method is not longer required.